server/app.py

```diff
@@ -5,26 +5,35 @@
 from api.login_handler import LoginResource
 from api.user_handler import UserResource
 from api.meal_item_handler import MealItemResource
+from api.LogoutResource import LogoutResource
+from api.TokenRefreshResource import TokenRefreshResource
 
-from api.ping_handler import ping_handler
-from api.home_handler import home_handler
-
-from flask_jwt_extended import JWTManager
 
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token,
+    jwt_refresh_token_required, create_refresh_token,
+    get_jwt_identity
+)
 from models import db
 
 
 def create_app():
     app = Flask(__name__)
+    app.config['JWT_TOKEN_LOCATION'] = ['cookies']
+
+    # Only allow JWT cookies to be sent over https. In production, this
+    # should likely be True
+    app.config['JWT_COOKIE_SECURE'] = False
+    app.config['JWT_ACCESS_COOKIE_PATH'] = '/'
```

Comment:
This is a future thing (maybe an icebox ticket) but do we want to have our whole API prefixed with /api?

I generally have found that easier, especially when we deploy

Reply:
done